lib/Extension/Bundle/Traits/PackageVersionTrait.php

```diff
@@ -27,12 +27,26 @@
  */
 trait PackageVersionTrait
 {
+
     /**
      * Returns the composer package name used to resolve the version
      *
      * @return string
      */
-    abstract protected function getComposerPackageName(): string;
+    public function getComposerPackageName(): string {
+        $composerFile = __DIR__ . '/../composer.json';
+
+        if (file_exists($composerFile)) {
+            $composerConfig = file_get_contents($composerFile);
+            $composerConfig = json_decode($composerConfig);
+
+            if (property_exists($composerConfig, 'name')) {
+                return $composerConfig->name;
+            }
+        }
```

Comment:
While the `*Bundle` class in Symfony's [current bundle structure](https://symfony.com/doc/4.4/bundles/best_practices.html#directory-structure) (which is not supported by Pimcore at the moment, though, but [I'm on it](https://github.com/pimcore/pimcore/issues/10912)) is located in the `src/` directory, this was not necessarily the case in [the old one](https://symfony.com/doc/3.4/bundles/best_practices.html#directory-structure). Some people put their code [into an `src/` folder](https://github.com/FriendsOfSymfony/FOSHttpCacheBundle) and [some don't](https://github.com/FriendsOfSymfony/FOSUserBundle). So I'd suggest supporting both:

```suggestion
        foreach([ __DIR__ . '/../composer.json',  __DIR__ . '/composer.json'] as $composerFile) {
            if (file_exists($composerFile)) {
                $composerConfig = json_decode(file_get_contents($composerFile));
    
                if (property_exists($composerConfig, 'name')) {
                    return $composerConfig->name;
                }
            }
        }
```